Gemfile.lock

```diff
@@ -261,4 +261,4 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
```

Comment:
these are all changes so it could merge into develop, even though: please don't, since there is a branch which really conflicts